devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -13,6 +13,9 @@
 #include <string_view>
 #include <unordered_map>
 #include <unordered_set>
+#include <atomic>
+#include <fstream>
+#include <jsoncpp/json/json.h>
```

Comment:
Leftover?

Reply:
Yes, fixed in [d92f7f4](https://github.com/ami-iit/bipedal-locomotion-framework/pull/796/commits/d92f7f4392f6d2c5b42a7494948cb755fe16bf80)